tools/run_tests/performance/scenario_config.py

```diff
@@ -830,9 +832,20 @@ def scenarios(self):
         rpc_type='STREAMING', client_type='SYNC_CLIENT', server_type='SYNC_SERVER',
         server_language='c++', async_server_threads=1)
 
-  def __str__(self):
-    return 'php'
+    yield _ping_pong_scenario(
+        '%s_to_cpp_protobuf_sync_unary_qps_unconstrained' % php_extension_mode,
+        rpc_type='UNARY', client_type='SYNC_CLIENT', server_type='ASYNC_SERVER',
+        server_language='c++', outstanding=1, async_server_threads=1, unconstrained_client='sync')
```

Comment:
I find that async_server_threads=1 is better than using default 0. async_server_threads=4 is the best.
Maybe there is some bottleneck during context switching? Since it uses cpp server, I don't want the qps result effected by something other than php. Thus set async_server_threads to a fix number can get a more accurate observation of php's bottleneck.

Reply:
64 client channels, streamming. If I understand right, `async_server_threads=0` means use as many threads as the server can.
qps/server_core result:
```
async_server_threads        0    1
c-proto-extension         5966   5049
php-proto-extension       3466   3491
```
This result is interesting.

Reply:
you can add a comment that normally async_server_threads is set to 0 for unconstrained scenarios, but for PHP  =1 performs better. (and perhaps a TODO to investigate).

Reply:
Done.

Reply:
I agree with Jan's suggestion.